07-reputation/04-globaltrust.md

```diff
@@ -0,0 +1,39 @@
+## Global Trust
+
+*Global trust*
+: the result of the [EigenTrust](http://ilpubs.stanford.edu:8090/562/1/2002-56.pdf) algorithm -- the trust in the network participant, which was obtained taking into account *all* local trusts of *all* nodes.
+
+### Subject and Object of a trust
+
+The Subject of the assessment is the entire system. Any node in the network is an Object of trust.
+
+The evaluation criterion is the aggregate of local trust of *all* network participants to the current node, but adjusted for similar local trust to each participant from the rest of the network.
+
+### Calculating trust
+
+Each manager retrieves the local trust value of its child at the end of each epoch and stores it in the `DaughterStorage`. The further task of each manager is to calculate the Global trust of the system in its child node, and also to transfer all the information he has to other managers so that they, in turn, can perform a similar task.
```

Comment:
.., and also to transfer all the information it has...

Reply:
fixed